Eu vou estudar mais

ftonato opened the pull request:
Eu vou estudar mais
Eu vou estudar mais
Eu vou estudar mais
![_](https://cloud.githubusercontent.com/assets/5417662/17160831/8d5bde88-537e-11e6-9132-536529f7e601.jpg)

Commit:
Adicionar dois novos cursos

Commit:
Adicionar a conclusão do mini-curso React.js ES6

Commit:
Adicionar a conclusão do curso Deploy a Website

Commit:
Adicionar estudo de React na Prática

Commit:
Adicionar dois novos cursos e correção da marcação

Adicionar dois mini-cursos: (Python e Flask, e Visão Geral de ERP)
Correção da marcação das palavras Ubuntu e Mint

Commit:
Adicionar mais tarefas/desafios

Commit:
Adicionar a conclusão da minha página pessoal

Commit:
Adicionar a conclusão do inglês um novo mini-curso

Adicionar a conclusão de 2 mêses de inglês

Commit:
Adicionar estudos de inglês do Inglês de Bolso

Commit:
Adicionar os vídeos do evento FrontInSP 2016

Commit:
Adicionar links para cursos

Adicionar links para cursos Python e Flask e Visão Geral de ERP.

Commit:
Adicionar dois mini-cursos de JS

leobalter commented:
@ftonato tem como mandar um git rebase no seu branch? Assim eu consigo dar merge no PR

Commit:
Corrigir conflitos

ftonato commented:
Deu trabalho mais foi :)

Agradecimento especial ao @gpedro e ao LINUX ♥

/cc @leobalter

ftonato committed:
Remover arquivo desnecessário!

Commit:
Adicionar novo estudo de inglês (Elllo)

Level 2 - Beginner Basics
Level 3 - High-Beginner
Level 4 - Low Intermediate
Level 5 - Intermediate
Level 6 - High Intermediate
Level 7 - Advanced English

woliveiras commented:
Boa @ftonato! 

Está arrebentando no inglês ein?

gpedro commented:
o @ftonato tá deitando no inglês
https://www.youtube.com/watch?v=h1MW77-q4LQ

ftonato commented:
Fala @woliveiras estou tentando aprender né, o problema é não conseguir mensurar o quão útil o conteúdo é, e "não treinar o que aprendo", por isso julgo precisar de um mentor, como já [vimos aqui](https://github.com/training-center/mentoria/issues/92).

@gpedro obrigado pelo vídeo, salvou minha noite <3 hahahahaha

woliveiras commented:
Verdade @ftonato 

Será que o @leobalter não consegue ajudar nessa ae? :smile:

Commit:
Adicionar a conclusão do mini-curso Jogo simples em JavaScript

leobalter commented:
Eu deito no inglês legal por aqui... Uma palavra pra pirar a gente: "Focus". O jeito que a gente pronuncia sai como se fosse "F*ck yous".
